CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.16)
+
+project(mozillavpn VERSION 2.9.0 LANGUAGES C CXX)
```

Comment:
2 things:
- is this used for the final binary file?
- can we remove the version.pri?

Reply:
Yes, this is used for the final binary.
We can remove `version.pri` as soon as we feel like dropping support for `qmake`

Reply:
Ok. Asking because the target changes for each platform. On MacOS and Windows it is "Mozilla VPN". On Linux "mozillavpn". Can we have it dynamic?